foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/ConfigStoreInit.java

```diff
@@ -93,31 +93,32 @@ public void init() {
     }
 
     private void pullConfigs() throws IOException {
```

Comment:
I: Small refactor to de-duplicate some code, improve log output when BASE_PROPERTY isn't set and ensure landing directories for extensions are always created even if the ENVIRONMENT_PROPERTY isn't set. This prevents the service from crashing if env overrides aren't provided.